.cicd/platforms/pinned/amazon_linux-2-pinned.dockerfile

```diff
@@ -9,15 +9,19 @@ RUN yum update -y && \
     libuuid-devel libtasn1-devel expect socat libseccomp-devel && \
     yum clean all && rm -rf /var/cache/yum
 # install erlang and rabbitmq
-RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh | bash && \
+RUN curl -fsSLO https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh && \
+    bash script.rpm.sh && \
+    rm script.rpm.sh && \
     yum install -y erlang
-RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh | bash && \
+RUN curl -fsSLO https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh && \
```

Comment:
You can have a function for  curl and run , and another one for curl and untar to avoid duplicate code.

Reply:
Can you have functions in docker files?

Reply:
I think this could be done by running a shell script that had the functions. That would require more work here than I think is warranted. The only reason to change the platform files at all was to originally fix an apt problem in one of the ubuntu 18.04 docker files which led me to find some other issues.